test/platform/scenes/spawn-point.spec.ts

```diff
@@ -0,0 +1,49 @@
+import expect from "expect";
+import { SpawnPoint } from "../../../src";
+import { testTypeSignature } from "../../test-utils";
+
+describe("Swap point tests", () => {
+  const swapPoint: SpawnPoint = {
```

Comment:
`swap` should be `spawn` right? Same for the title in the line above

Reply:
Yeah, good catch